azdev/operations/linter/linter.py

```diff
@@ -139,6 +150,8 @@ def command_group_expired(self, command_group_name):
             deprecate_info = group_kwargs.get('deprecate_info', None)
             if deprecate_info:
                 return deprecate_info.expired()
+        except KeyError:
```

Comment:
could you add comment for the scenario we get `KeyError`?

Reply:
```suggestion
        # ignore command_group_name which is not in command_group_table.
        except KeyError:
```